src/Middleware/SpaServices.Extensions/src/SpaOptions.cs

```diff
@@ -69,6 +70,27 @@ public PathString DefaultPage
         /// </summary>
         public string SourcePath { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name of the package manager executible, (e.g npm,
+        /// yarn) to run the SPA.
+        /// 
+        /// If not set, npm will be assumed as the default package manager 
+        /// executable
+        /// </summary>
+        public string PackageManagerName
```

Comment:
Should this really be `PackageManagerCommand` or something like that? It's not really the *name*, as the name might not be the same as the command to run.

Reply:
Agreed, changed to `PackageManagerCommand `